main.css

```diff
@@ -0,0 +1,623 @@
+}
+
+#logo {
+  align-self: flex-start;
+}
+
+main {
+  margin-top: 72px;
+  width: 100%;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: auto;
+  padding-top: 6.2em;
+  padding-bottom: 3.1em;
+  align-items: center;
+  margin: 0 auto;
+}
+
+#headline {
+  background: url(./pics/header-illsutration-mobile.png);
+  background-repeat: no-repeat;
+}
+
+h1 {
+  font-size: 2.5em;
+  width: 87.2%;
+  line-height: 3.2rem;
+  height: auto;
+  font-weight: 700;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: #172b4d;
+  margin: 0 auto;
+}
+
+#title-span {
+  color: #36b37e;
+}
+
+.text {
+  font-weight: 400;
+  text-align: center;
+  color: #172b4d;
+  font-size: 1.21rem;
+  line-height: 1.8rem;
+  margin-top: 3.6rem;
+  width: 87.2%;
+  height: auto;
+}
+
+.socials {
+  display: flex;
+  align-items: center;
+  height: auto;
+  width: 45.33%;
+  margin-top: 3.625rem;
+  justify-content: center;
+}
+
+li {
+  list-style: none;
+  text-decoration: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.work-row {
+  width: 88.26%;
+  padding: 4px 2px;
+  margin-top: 30px;
+  display: grid;
+  gap: 1.5em;
+  justify-content: center;
+}
+
+#recent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#indicator {
+  border-radius: 1.5em;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  height: 4px;
+  width: 3em;
+  background: #36b37e;
+}
+
+.item {
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+  border: 1px solid #d0d9d4;
+  width: 99.79%;
+  border-radius: 0.5em;
+}
+
+.placeholder {
+  width: 100%;
+}
+
+.text-card {
+  padding: 1em;
+  height: 222px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  background: #fff;
+  border-radius: 0.5em;
+  margin-top: -222px;
+}
+
+.subtitle {
+  height: auto;
+  width: 90.5%;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  color: #3a4a42;
+}
+
+h3 {
+  font-size: 1em;
+  font-weight: 600;
+  line-height: 2em;
+}
+
+.courses {
+  width: 19em;
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  align-self: center;
+}
+
+.courses li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600;
+  color: #3a4a42;
+  background: #ebf0ee;
+  border-radius: 4px;
+}
+
+button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  padding: 12px;
+  border-radius: 4px;
+  border: none;
+  background: #36b37f;
+  color: #fff;
+  font-weight: 500;
+  font-size: 1em;
+  line-height: 2em;
+  cursor: pointer;
+  transition: all 200ms linear;
+}
+
+button:hover {
+  background: #c5e9da;
+  box-shadow: 0 0.25em 0.25em;
+  opacity: 0.8;
+  color: #172b4d;
+}
```

Comment:
While I admire the effort you have put in to design the button interactions 👏👏👏👏 kindly try to stick to the Figma design since those interactions look much more professional 🤗

![image](https://user-images.githubusercontent.com/73714615/187036310-1a386794-a3da-4417-b57e-32c2859539b3.png)